### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by wyznerd** [jmoor3/homeassistant-components#1]
I'm a bit confused on how to actually get this running as a custom_component.  If I add it simply as a component in my config directory, HA seems to completely ignore it.  If I add it as a custom_component, I can't get spaclient to import as a util.

Any thoughts on what I actually need to do?

**Comment by jmoor3** [comment]
You need to add it to the main components directory. I haven't figured out how to add a custom light. Are you adding things to the directories I listed in the readme?

**Comment by wyznerd** [comment]
Yeah, I added them to the main components directory but then get errors within import.

**Comment by jmoor3** [comment]
Are you adding to the utils folder that is one level up from components? What are your import errors?

**Comment by jmoor3** [comment]
I changed it to work out of the custom_components folder, which should make this easier.
